src/app/docs-pages/selector-doc/selector-doc.component.html

```diff
@@ -4,12 +4,25 @@
     <h1 class="doc-title">Selector / Dropdown component</h1>
   
     <p class="doc-info">
+      Dropdowns are useful for organizing and showing the user a filtered selection.
+    </p>
+
+    <div class="doc-info-use">
+      <h2><i class="fal fa-check-circle doc-info-use-icons icon-green"></i> <b>When to use:</b></h2>
+      <p>
         A dropdown should be used if the user wants to make a selection from a set of predefined choices. 
         The recommended selection for the user should be selected as default. 
         If the list of choices is longer than 5, then an explanatory placeholder text should be used.
         The label should clearly describe the action, and the placeholder text should not consist of any additional information compared to the label. 
-        This as the placeholder is only visible before making a selection.
-    </p>
+        This as the placeholder is only visible before making a selection. 
+      </p>
+    </div>
+    <div class="doc-info-use">
+      <h2><i class="fal fa-times-circle doc-info-use-icons icon-red"></i> <b>When not to use:</b></h2>
+      <p>
+        A Dropdown should not be used when there are too many or too few options.
```

Comment:
Alt for vag og uspesifikk. For noen er for mye 100000, for andre er det 20

Reply:
om x< 3 så borde det vara en checkbox eller radio button beroende på vad man ska göra. Å andra sidan så är det vanskeligt att säga hur mycket som är för mycket. Är det 100000 så borde man åtminstone kunna filtrera och söka i tillägg

Reply:
"This Dropdown should not be used when there are less than 3 alternatives or the numbers of alternatives are so large its becoming hard to find the right choice."